libcloud/backup/drivers/dimensiondata.py

```diff
@@ -384,6 +421,73 @@ def cancel_target_job(self, target, job):
         raise NotImplementedError(
             'cancel_target_job not implemented for this driver')
 
+    def ex_add_client_to_target(self, target, client, storage_policy,
+                                schedule_policy, trigger, email):
+        """
+        Add a client to a target
+
+        :param target: Backup target with the backup data
+        :type  target: Instance of :class:`BackupTarget` or ``str``
+
+        :param client: Client to add to the target
+        :type  client: ``str``
+
+        :param storage_policy: The storage policy for the client
+        :type  storage_policy: ``str``
+
+        :param schedule_policy: The storage policy for the client
+        :type  schedule_policy: ``str``
+
+        :param trigger: The notify trigger for the client
+        :type  trigger: ``str``
+
+        :param email: The notify email for the client
+        :type  email: ``str``
+
+        :rtype: ``bool``
+        """
+
+        if isinstance(target, BackupTarget):
+            server_id = target.address
+        else:
+            server_id = target
+
+        backup_elm = ET.Element('NewBackupClient',
+                                {'xmlns': BACKUP_NS})
+        ET.SubElement(backup_elm, "type").text = client
+        ET.SubElement(backup_elm, "storagePolicyName").text = storage_policy
```

Comment:
Is the client code to fetch the storage policies and schedule policies implemented? It would be preferable to send the storagepolicy response from that client call instead of the name as a string.